content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/harbor/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "How to Sync Images from Chainguard's Registry to Harbor"
+linktitle: "Harbor"
+type: "article"
+description: "Tutorial outlining how to sync images from Chainguard's registry to Harbor."
+date: 2025-08-19T12:00:00-00:00
+lastmod: 2025-08-19T12:00:00-00:00
+draft: false
+tags: ["Chainguard Containers"]
+images: []
+menu:
+  docs:
+    parent: "pull-through-guides"
+toc: true
+weight: 015
+---
+
+[Harbor](https://goharbor.io) is an open-source artifact registry. It's designed to securely store, manage, and distribute OCI artifacts, including container images and Helm charts by enforcing policies like vulnerability scanning, image signing, and role-based access control. Harbor delivers enterprise-grade compliance, performance, and interoperability across platforms like Kubernetes and Docker, all accessible via a web UI or RESTful API.
+
+This tutorial outlines how to sync images from Chainguard's registry to a Harbor instance. It describes two approaches:
+
+1. A [proxy cache](https://goharbor.io/docs/2.1.0/administration/configure-proxy-cache/), which configures a Harbor project as a pull through cache.
+2. A [replication rule](https://goharbor.io/docs/2.1.0/administration/configuring-replication/create-replication-rules/), which copies images to a Harbor project.
+
+## Prerequisites
+
+You need the following in order to complete this tutorial:
+
+* Administrative privileges over a Harbor instance. Refer to the [official Harbor documentation](https://goharbor.io/docs/2.13.0/) to learn how to set this up.
+* `chainctl` — Chainguard's command-line interface — installed on your local machine. If you don't have `chainctl` installed, refer to our [How to Install `chainctl`](/chainguard/chainctl-usage/how-to-install-chainctl/) guide to set this up.
+* Permissions pull Chainguard container images from your organization's repository within the Chainguard registry. This is necessary, as you will create a pull token for Harbor to use to access the registry, and you cannot generate a pull token that grants broader access than your own.
```

Comment:
"permissions *to* pull"

But maybe maybe rephrase e.g. "Access to an account with permissions to pull container images...."